stacks/epinio/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+./"${SCRIPT_DIR}/common.sh"
```

Comment:
it doesn't work on my env:
`./deploy.sh: line 6: .//home/juadk/git/marketplace-kubernetes/stacks/epinio/common.sh: No such file or directory`
```suggestion
source "${SCRIPT_DIR}/common.sh"
```

Reply:
`source` was giving me a warning from shellcheck:

```
In deploy.sh line 6:
source "${SCRIPT_DIR}/common.sh"
^-- SC3046 (warning): In POSIX sh, 'source' in place of '.' is undefined.
       ^-----------------------^ SC1091 (info): Not following: ./common.sh was not specified as input (see shellcheck -x).
```

Reply:
and what about:
`. /"${SCRIPT_DIR}/common.sh"`

Reply:
yes shellcheck only accepts `./"${SCRIPT_DIR}/common.sh"`
Don't know why it fails on my env though.